source/_components/frontend.markdown

```diff
@@ -18,6 +18,13 @@ This offers the official frontend to control Home Assistant.
 frontend:
 ```
 
+Configuration variables:
```

Comment:
Isn't there some new format for config variables?

Reply:
There is but at the moment it's more do it quick than do it nice. As I need to update it anyways, I can add it.